From 21a106ea5453c62be1a59676656d04bf63cb33cf Mon Sep 17 00:00:00 2001
From: Kishan Mochi <kishan.mochi@intel.com>
Date: Fri, 24 Sep 2021 10:46:50 +0530
Subject: [PATCH] port change from horizontal to vertical for gen11

port 5afaba7 to vertical branch for gen11 media

Tracked-On: OAM-99591
Signed-off-by: Kishan Mochi <kishan.mochi@intel.com>

diff --git a/groups/codecs/configurable/media_codecs_gen11.xml b/groups/codecs/configurable/media_codecs_gen11.xml
index 255e663..50685d3 100644
--- a/groups/codecs/configurable/media_codecs_gen11.xml
+++ b/groups/codecs/configurable/media_codecs_gen11.xml
@@ -232,6 +232,9 @@ Only the three quirks included above are recognized at this point:
         </MediaCodec>
 {{/hw_ve_h265}}
     </Encoders>
+{{#sw_omx_video}}
+    <Include href="media_codecs_google_video.xml" />
+{{/sw_omx_video}}
     <Include href="media_codecs_google_audio.xml" />
     <Settings>
         <Setting name="max-video-encoder-input-buffers" value="9" />
--
2.33.0

